tasks/sshd.yml

```diff
@@ -56,4 +56,5 @@
   selinux_permissive:
     name: sshd_t
     permissive: true
-  when: ansible_selinux|bool
+  when: ( ansible_selinux is defined and ( (ansible_selinux is mapping and ansible_selinux.status == 'enabled') or ( ansible_selinux|bool ) )  )
+  notify: Restart sshd
```

Comment:
Goes a little bit "hidden" and unmentioned in the comment. Maybe better in its own commit?

Reply:
ok